solr/core/src/java/org/apache/solr/cloud/ZkController.java

```diff
@@ -323,22 +329,29 @@ public ZkController(
     ZkClientConnectionStrategy strat =
         ZkClientConnectionStrategy.forName(connectionStrategy, new DefaultConnectionStrategy());
 
+    String zkCredentialsInjectorClass = cloudConfig.getZkCredentialsInjectorClass();
+    ZkCredentialsInjector zkCredentialsInjector =
+        !StringUtils.isEmpty(zkCredentialsInjectorClass)
```

Comment:
I'm ok with the ternary operator. But for these 3 checks, could you remove the `!` and swap the true/false conditions? It will make it easier to read.